apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -106,6 +106,27 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
+        invalidServicesError:  {
+            schemaDoc: 'Indicates if services are valid and add up to total sum toPay.',
+            type: Virtual,
+            isRequired: false,
+            resolver: async (item) => {
+                let servicesTotal = 0
```

Comment:
I think its better to initialize it as `Big(0)` to indicate, that you are going to do some calculations here